auth_supplier/views/auth_supplier_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="auth_supplier.fields"
+              inherit_id="auth_signup.fields"
+              name="Auth Signup/ResetPassword form fields">
+        <xpath expr="//div[@class='form-group field-login']" position="before">
+            <div class="form-group field-type">
+                <label for="account_type" class="control-label">Account Type</label>
+                <select class="form-control" name="account_type">
+                    <option value="customer" selected="selected">Customer</option>
```

Comment:
I tested the 'Customer' option as it seems to be the default behaviour (auth_signup), but the resulting partner is not a customer (not checked). Should we rename this option 'Portal user', 'Default', 'User' or something else?